packages/fx-core/src/common/samples-config.json

```diff
@@ -42,6 +42,16 @@
             "configuration": "Ready for local debug",
             "suggested": true
         },
+        {
+            "id": "graph-toolkit-one-productivity-hub",
+            "title": "One Productivity Hub using Graph Toolkit",
+            "shortDescription": "One Productivity Hub app of Microsoft Teams Tab app.",
+            "fullDescription": "One Productivity Hub sample shows you how to build a tab for viewing your calendar events, to-do tasks and files by using Microsoft Graph Toolkit as UI component.",
+            "tags": ["Tab", "JS"],
+            "time": "5min to run",
+            "configuration": "Ready for local debug",
+            "suggested": true
```

Comment:
Is this a suggested sample by PM/UX?

Reply:
@zyxiaoyuer , what do think about this?

Reply:
by Suggested, do you mean the star? 
BTW. I remeber there is a SSO tag also.

Reply:
Thanks, updated suggested to false, and add sso tag